src/include/btmem.h

```diff
@@ -1121,6 +1121,21 @@ struct __wt_update {
  */
 #define WT_UPDATE_MEMSIZE(upd) WT_ALIGN(WT_UPDATE_SIZE + (upd)->size, 32)
 
+/*
+ * WT_UPDATE_VIEW --
```

Comment:
Not sure what a good name for this is.

Reply:
Yeah, I'm with you here, I don't much like `WT_UPDATE_VIEW`. Hmmm. `WT_CURSOR_BTREE_VALUE`?

Reply:
Ended up calling it `WT_UPDATE_VALUE`. I think the `CURSOR_BTREE` part would be redundant since it's part of the `cbt` already.